pkg/serviceprovider/snyk/tokenfilter.go

```diff
@@ -0,0 +1,17 @@
+package snyk
+
+import (
+	"context"
+
+	api "github.com/redhat-appstudio/service-provider-integration-operator/api/v1beta1"
+	"github.com/redhat-appstudio/service-provider-integration-operator/pkg/serviceprovider"
+)
+
+type tokenFilter struct{}
+
+var _ serviceprovider.TokenFilter = (*tokenFilter)(nil)
+
+func (t *tokenFilter) Matches(_ context.Context, matchable serviceprovider.Matchable, token *api.SPIAccessToken) (bool, error) {
+	//TODO: implement token matching
```

Comment:
We don't have anything to do the matching with, so I guess this really is the implementation. No need to leave the TODO here, maybe just an explanation why we're simply always matching.